### PR TITLE
Added Lists link to books page navbar

### LINKED ISSUE #6096: Add "Lists" link to books page navbar
### Describe the problem that you'd like solved

When visiting a books page, I have to scroll to nearly the bottom of the page to see the lists that contain the book.  It would be great if I could jump to that section by clicking a link on the navbar.

### Proposal & Constraints

Add a new link to the book page navbar that will jump to the "Lists containing this Book" section of the page.  A unique `id` will need to be added to that section.  Ensure that the new `id` is written in [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case), if necessary.

The text for the link should be "Lists", and it should be internationalized.  The new link should appear rightmost on the navbar.

### PR DESCRIPTION
**Pull request by homewardgamer**
Closes #6096

Added `Lists` link to books page navbar after the `Reviews` link.

### Technical

Added the id `lists` to the  [Lists containing this Book](https://openlibrary.org/works/OL16225732W/Life's_Handicap/lists) section.

### Screenshot

Screenshot 2022-02-03 at 12 21 47 PM

### Stakeholders

@jimchamp  @mekarpeles

### CODE REVIEW

File: openlibrary/templates/type/edition/view.html
Comment (mekarpeles) [inline openlibrary/templates/type/edition/view.html:481]: I wonder if we want to use `name` here instead of `id` to avoid any possible styling collisions. @jimchamp ?

Otherwise, changes LGTM!

Comment (jimchamp) [reply openlibrary/templates/type/edition/view.html:481]: I think that using `id` is fine.  I didn't see any CSS rules for `#lists` after doing a quick search, but it may be better to be more specific with the ID.  Something like `lists-section` or `lists-details` would be great.  I'll let you decide which, @homewardgamer.

**Review by jimchamp — CHANGES_REQUESTED** [review]
Happy to merge this once the ID has been updated.

**Comment by homewardgamer** [comment]
@jimchamp  Changed the id name to `lists-section`.

**Review by jimchamp — APPROVED** [review]
Lgtm!

**Pull request merged**
